build_scripts/webpack.build.js

```diff
@@ -75,25 +73,19 @@ module.exports = {
   },
   plugins: [
     new ForkTsCheckerWebpackPlugin({ checkSyntacticErrors: true }),
-    new webpack.optimize.UglifyJsPlugin({
```

Comment:
this plugin is chronically unhappy with the output of `ts-loader`. I imagine it's also not entirely necessary given that folks can/will be minifying their app builds.